docs/rst/tutorials/cloud/router_conf_via_xml_profiles/router_conf_via_xml_profiles.rst

```diff
@@ -0,0 +1,165 @@
+.. include:: ../../../exports/alias.include
+
+.. _tutorials_cloud_xml_participant_configuration:
+
+ROS 2 Router configuration via XML profiles
+===========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+
+Background
+----------
+
+*eProsima ROS 2 Router*, a.k.a `DDS Router <https://github.com/eProsima/DDS-Router>`_, is an end-user software application that enables the connection of distributed ROS 2 networks (see the |rosrouter| documentation :ref:`here <vulcanexus_router>`).
+That is, ROS 2 nodes such as publishers and subscriptions, or clients and services, deployed in one geographic location and using a dedicated local network will be able to communicate with other ROS 2 nodes deployed in different geographic locations on their own dedicated local networks as if they were all on the same network through the use of |rosrouter|.
+
+This tutorial explains how to configure a Participant with XML. In particular, we will configure two Participants, one without XML on domain ``0`` and one with XML on domain ``1``, and use the |rosrouter| to allow them to communicate between each other.
+
+.. note::
+
+    This tutorial is similar to the :ref:`tutorials_router_change_domain` tutorial, since we will launch a *talker* and a *listener* on different domains and connect them with the |rosrouter|. The main difference between both tutorials is that, in this one, one of the Participants' configurations will be written in XML.
```

Comment:
```suggestion
    This tutorial is similar to the :ref:`tutorials_router_change_domain` tutorial, since we will launch a *talker* and a *listener* on different domains and connect them with the |rosrouter|. The main difference between both tutorials is that, in this one, one of the Participants will be configured using Fast DDS XML profiles.
```